### PR TITLE
Release to PRD

### PR DESCRIPTION
**Pull request by bitbeckers**
* Pagination improvements
* Improved suspense and usage of server-client components on evaluators and explore page
* Celo marketplace support
* Decimal inputs in allow lists
* Transfer, split, merge, burn flows

### CODE REVIEW

File: components/currency-buttons.tsx [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 
-export const CurrencyButtons = ({ className }: { className?: string }) => {
-  const pathname = usePathname();
-  const searchParams = useSearchParams();
+interface CurrencyButtonsProps {
+  value?: string;
+  className?: string;
+}
+
+export const CurrencyButtons = ({ value, className }: CurrencyButtonsProps) => {
```
Comment (Jipperism) [inline components/currency-buttons.tsx:13]: Value prop isn't being passed on user profile when viewing marketplace orders so button doesn't show correct state.

File: components/evaluators/evaluators-list.tsx [hunk 2]
```diff
@@ -1,34 +1,39 @@
...
+}
+
+export default function EvaluatorsList({
   searchParams,
 }: {
   searchParams: Record<string, string>;
 }) {
-  const attestors = await getTrustedAttestors();
   const currentPage = Number(searchParams?.p) || 1;
+  const attestors = use(getAttestorsData());
```
Comment (Jipperism) [inline components/evaluators/evaluators-list.tsx:19]: `use` seems to be a canary/preview only function, FYI. I'm surprised it's available as we don't seem to be on either of those versions of react.

https://react.dev/reference/react/use

Comment (bitbeckers) [reply components/evaluators/evaluators-list.tsx:19]: Probably NextJS team already implemented exposes it: https://nextjs.org/blog/next-14-2

File: components/currency-buttons.tsx (hunk 1 above)
Comment (bitbeckers) [reply components/currency-buttons.tsx]: Good catch, thanks! Fixed in #311

**Pull request merged**